scripts/Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM jderobot/base:3.2
 
+
+
 # Custom Robot Repository
 RUN mkdir -p /opt/jderobot && cd /opt/jderobot && \
 	git clone -b $ROS_DISTRO-devel https://github.com/JdeRobot/CustomRobots.git
 
+
+RUN pip install django-webpack-loader djangorestframework
```

Comment:
Why is this needed?

Reply:
solves the following error:
![error_RADI](https://user-images.githubusercontent.com/66624139/179525869-9e5212b8-46b4-4b0e-894e-efe3ceb8c0f0.png)

Reply:
asked on the main RA channel, waiting for better suggested solution